parsons/airmeet/airmeet.py

```diff
@@ -0,0 +1,415 @@
+                The id of the Airmeet.
+            lists_to_tables: bool
+                If True, will convert any dictionary values that are lists
+                to Tables.
+        `Returns:`
+            Dict containing the Airmeet data
+        """
+        response = self.client.get_request(url=f"airmeet/{airmeet_id}/info")
+        if lists_to_tables:
+            for k in response:
+                if isinstance(response[k], list):
+                    response[k] = Table(response[k])
+        return response
+
+    def fetch_airmeet_custom_registration_fields(self, airmeet_id) -> Table:
+        """
+        Get the list of custom registration fields for an Airmeet.
+
+        `Args:`
+            airmeet_id: string
+                The id of the Airmeet.
+        `Returns:`
+            Parsons.Table
+                List of custom registration fields for this Airmeet event
+        """
+        response = self.client.get_request(url=f"airmeet/{airmeet_id}/custom-fields")
+        return Table(response["customFields"])
+
+    def fetch_event_attendance(self, airmeet_id) -> Table:
+        """
+        Get all attendees for an Airmeet, handling pagination based on the
+        returned pageCount.
+
+        Results include attendance only from sessions with a status of
+        `FINISHED`. Maximum number of results per page = 50.
+
+        "This is an Asynchronous API. If you get a 202 code in response,
+        please try again after 5 minutes."
+
+        `Args:`
+            airmeet_id: string
+                The id of the Airmeet.
+        `Returns:`
+            Parsons.Table
+                List of attendees for this Airmeet event
+        """
+        return self._get_all_pages(url=f"airmeet/{airmeet_id}/attendees", page_size=50)
+
+    def fetch_session_attendance(self, session_id) -> Table:
+        """
+        Get all attendees for a specific Airmeet session, handling pagination
+        based on the returned pageCount.
+
+        Results are available only for sessions with a status of `FINISHED`.
+        Maximum number of results per page = 50.
+
+        "This is an Asynchronous API. If you get a 202 code in response,
+        please try again after 5 minutes."
+
+        `Args:`
+            session_id: string
+                The id of the session.
+        `Returns:`
+            Parsons.Table
+                List of attendees for this session
+        """
+        return self._get_all_pages(url=f"session/{session_id}/attendees", page_size=50)
+
+    def fetch_airmeet_booths(self, airmeet_id) -> Table:
+        """
+        Get the list of booths for a specific Airmeet by ID.
+
+        `Args:`
+            airmeet_id: string
+                The id of the Airmeet.
+        `Returns:`
+            Parsons.Table
+                List of booths for this Airmeet
+        """
+        response = self.client.get_request(url=f"airmeet/{airmeet_id}/booths")
+        return Table(response["booths"] or [])
```

Comment:
I'm curious about this syntax - does `response["booths"]` return something weird like None?

Reply:
@shaunagm Yes, if there are no booths for the Airmeet, the API returns `{"booths":null}`